castai/resource_castai_cluster.go

```diff
@@ -100,6 +107,128 @@ func resourceCastaiCluster() *schema.Resource {
 				},
 			},
 			ClusterFieldKubeconfig: schemaKubeconfig(),
+			PolicyFieldAutoscalerPolicies: {
```

Comment:
Some fields are constants and some are inline which is not consistent. Since there is a lot of fields and used only in few places I would remove this constants and inline all schema fields as strings.